src/sentences/ttm.rs

```diff
@@ -11,6 +12,7 @@ use serde::{Deserialize, Serialize};
 
 use super::utils::{parse_float_num, parse_hms, parse_number_in_range};
 use crate::{Error, NmeaSentence, SentenceType};
+use heapless;
```

Comment:
that's actually not needed  as an import. I would suggest importing `heapless::String` directly.